src/core/Form/FilterInput/FilterInput.tsx

```diff
@@ -67,8 +69,12 @@ interface InternalFilterInputProps<T>
   onFilter: (filteredItems: Array<T>) => void;
```

Comment:
The comment on this one seems out of place.

Reply:
Yes and no, it's a callback for filtering event and returns the filtered items when changed. This is an old comment, but will improve it anyway.

Reply:
Fixed to indicate that this is a callback that gets new filtered items as a parameter.